console/src/main/scripts/plugins/accessPortal/vendor/js/custom-cases.js

```diff
@@ -1,10 +1,12 @@
+var attachmentsRequest = 'hawkular-redhat-access-integration-backend/attachments';
+
```

Comment:
A javascript project and not typescript? All of the other plugins are typescript.